apollo-compiler/src/test/graphql/com/example/pojo_custom_scalar_type/TestQueryExpected.java

```diff
@@ -161,10 +171,14 @@ public void handle(final int fieldIndex, final Object value) throws IOException
                   contentValues.appearanceDates = (List<? extends Date>) value;
                   break;
                 }
+                case 3: {
+                  contentValues.fieldWithUnsupportedType = (Object) value;
+                  break;
+                }
```

Comment:
agh, well this code is generated, so we guarantee that there won't be default case